_sass/_print.scss

```
@@ -0,0 +1,79 @@
+  }
+
+  // Don't annotate internal/anchor links
+  .post-body a[href^="/"]::after,
+  .post-body a[href^="#"]::after {
+    content: none;
+  }
+
+  // Keep images within page
+  img {
+    max-width: 100%;
+    page-break-inside: avoid; // legacy
```

Stylelint (17.4.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)